src/devtools/client/debugger/src/components/SecondaryPanes/EventListeners.js

```diff
@@ -313,11 +318,9 @@ class EventListeners extends Component {
               {searchText ? this.renderCategory(category) : null}
               {event.name}
             </span>
-            {features.eventCount ? (
```

Comment:
what is eventCount supposed to be doing here? It looks like now in the case where `features.eventCount` is false but there are `points` we will still render all of this, is that right?

Reply:
yeah - i think it's safe to remove this